code/modules/reagents/chemistry/machinery/chem_heater.dm

```diff
@@ -37,7 +37,7 @@
 			if(!beaker.reagents.total_volume)
 				on = FALSE
 				return
-			beaker.reagents.temperature_reagents(desired_temp, max(1, 35 - speed_increase))
+			beaker.reagents.change_reagent_temp(speed_increase, desired_temp)
```

Comment:
Since the first argument now decides the direction the temperature is being adjusted in, the heater will now need to calc it on its own
```suggestion
			var/sign = SIGN(desired_temp - beaker.reagents.chem_temp)
			beaker.reagents.change_reagent_temp(speed_increase * sign, desired_temp)
```